packages/fluentui/accessibility/src/behaviors/Card/cardBehavior.ts

```diff
@@ -0,0 +1,18 @@
+import { Accessibility } from '../../types';
+/**
+ * @description
+ * Behavior for a card component - semantic grouping of objects
+ * @specification
+ * Adds role `group` to the container
```

Comment:
```suggestion
 * Adds role='group'.
```

Reply:
There are automatic test for the behaviors based on the specification. Make sure you are using valid expressions here.